profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/tmgmt_poetry.module

```diff
@@ -738,6 +750,121 @@ function tmgmt_poetry_page_alter(&$page) {
 /**
  * Implements hook_form_FORM_ID_alter().
  *
+ * Change the #cart form to provide the tmgmt_poetry module functionalities.
+ */
+function tmgmt_poetry_form_tmgmt_ui_cart_content_alter(&$form, &$form_state) {
+  $form['request_translation']['#submit'] = ['_tmgmt_poetry_workbench_store_request_languages_callback'];
+}
+
+/**
+ * Change the #cart submit logic to provide tmgmt_poetry module functionalities.
+ *
+ * @see tmgmt_ui_cart_request_translation_form_submit()
+ */
+function _tmgmt_poetry_workbench_store_request_languages_callback(&$form, &$form_state) {
```

Comment:
It's because this function is replacing the original "tmgmt_ui_cart_request_translation_form_submit".
I just want to change the minimal code here, so that it can be better compared, what we are tweaking.

But I will put this code into the new module tmgmt_dgt_connector, it's all about the cart/small jobs.